core/src/main/java/org/apache/calcite/util/Util.java

```diff
@@ -1048,6 +1048,19 @@ public static String getStackTrace(Throwable t) {
     return sw.toString();
   }
 
+  /**
+   * Checks a RuntimeException for AccessControlException without importing JDK
+   * classes that are deprecated with the anticipated removal of Java security manager.
+   * Ignores the RuntimeException if it is an AccessControlException otherwise throws.
+   * @param e RuntimeException to check if it is an AccessControlException
+   */
+  @API(since = "1.33", status = API.Status.EXPERIMENTAL)
+  public static void ignoreAccessControlException(RuntimeException e) {
```

Comment:
So I thought about trying to put the whole `try/catch` in here but that got into issues with return values and handling other exceptions. There may be a way to do this with lambdas or functions passed in. I just punted on that and made this a simple method just for checking `AccessControlException`

Reply:
Is this issue tied to any particular JDK version? E.g. SecurityManager became deprecated in JDK x.x and is going to be removed in JDK y.y? If so it would be helpful to mention in this comment.

Reply:
Fixed in ede956c7aecf560ea88c45b48ef7a351f0c8e891